libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -310,11 +311,13 @@ string IRGeneratorForStatements::constantValueFunction(VariableDeclaration const
 		string functionName = IRNames::constantValueFunction(_constant);
```

Comment:
Is calling `setLocation` above useful?

Reply:
hmm probably not as the actual general is delayed until the function collector starts.. removed it.